Terraform UBI agent fails to build with latest base image

**Describe the bug**
Terraform build with Dockerfile.ubi8 fails to build due to the latest changes in Jenkins base agent

**To Reproduce**
Steps to reproduce the behavior:
1. Build Terraform with Dockerfile.ubi8
2. See error

**Expected behavior**
The agent should build successfully

**Screenshots**
![image](https://user-images.githubusercontent.com/26645694/190361004-df9b712b-2bcd-4b37-b21f-c235683a1c9c.png)

**Affected version:**
 - OpenDevStack: master and 4.x

**Log Output**
```
Step 45/47 : RUN ls $JAVA_HOME/lib/security/cacerts     && chown 1001:0 $JAVA_HOME/lib/security/cacerts     && chmod g+w $JAVA_HOME/lib/security/cacerts
 ---> Running in 8e6914fc3ede
ls: cannot access '/lib/security/cacerts': No such file or directory
The command '/bin/bash -o pipefail -c ls $JAVA_HOME/lib/security/cacerts     && chown 1001:0 $JAVA_HOME/lib/security/cacerts     && chmod g+w $JAVA_HOME/lib/security/cacerts' returned a non-zero code: 2

Error: Process completed with exit code 2.
```

**Additional context**
This is due to JAVA_HOME being removed in base image
Related to https://github.com/opendevstack/ods-core/pull/1181

@victorpablosceruelo

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -65,6 +65,7 @@
 - Update nodejs version in TypeScript Quickstarter ([#834](https://github.com/opendevstack/ods-quickstarters/issues/834))
 - Fix failing acceptance test in cypress quickstarter ([#840](https://github.com/opendevstack/ods-quickstarters/issues/840))
 - Fix nodejs12 build fails with redhat jenkins agent ([#843](https://github.com/opendevstack/ods-quickstarters/issues/843))
+- Fix Build Terraform UBI agent fails ([#847](https://github.com/opendevstack/ods-quickstarters/issues/847))
 
 ## [4.0] - 2021-11-05
 

--- a/common/jenkins-agents/terraform/docker/Dockerfile.centos7
+++ b/common/jenkins-agents/terraform/docker/Dockerfile.centos7
@@ -130,7 +130,7 @@ RUN wget -q -O /tmp/parallel-20160222-1.el7.noarch.rpm   https://download-ib01.f
     && rpm -Uvh /tmp/parallel-20160222-1.el7.noarch.rpm \
     && yum install -y parallel \
     && yum clean all
-    
+
 #Install jq
 RUN yum -y install https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/e/epel-release-7-14.noarch.rpm \
     && yum install -y jq \
@@ -142,7 +142,7 @@ RUN wget -q "https://releases.hashicorp.com/consul/${CONSUL_VERSION}/consul_${CO
     && unzip consul_${CONSUL_VERSION}_linux_amd64.zip -d /usr/local/bin \
     && rm -f consul_${CONSUL_VERSION}_linux_amd64.zip \
     && chmod +x /usr/local/bin/consul && /usr/local/bin/consul -version
-    
+
 # Install mozilla/sops and AGE
 RUN yum install -y https://github.com/mozilla/sops/releases/download/v${SOPS_VERSION}/sops-${SOPS_VERSION}-1.x86_64.rpm \
     && wget -q -O /tmp/age.tar.gz https://github.com/FiloSottile/age/releases/download/v${AGE_VERSION}/age-v${AGE_VERSION}-linux-amd64.tar.gz \
@@ -159,15 +159,11 @@ COPY Gemfile Gemfile.lock $GEM_HOME/
 RUN chown -R 1001:0 $GEM_HOME \
     && chmod -R g+rw $GEM_HOME \
     && ls -lisa /home/jenkins $GEM_HOME
-    
-RUN ls $JAVA_HOME/lib/security/cacerts \
-    && chown 1001:0 $JAVA_HOME/lib/security/cacerts \
-    && chmod g+w $JAVA_HOME/lib/security/cacerts
 
 # setup ruby env and bundler gems
 # RUBY https://syslint.com/blog/tutorial/how-to-install-ruby-on-rails-with-rbenv-on-centos-7-or-rhel-7/
 RUN cd /opt \
-    && umask 0002 \ 
+    && umask 0002 \
     && git clone https://github.com/rbenv/rbenv.git /opt/rbenv \
     && echo 'export PATH="/opt/rbenv/shims:/opt/rbenv/bin:$PATH"' >> ~/.bash_profile \
     && echo 'eval "$(rbenv init -)"' >> ~/.bash_profile \

--- a/common/jenkins-agents/terraform/docker/Dockerfile.ubi8
+++ b/common/jenkins-agents/terraform/docker/Dockerfile.ubi8
@@ -138,7 +138,7 @@ RUN wget -q "https://releases.hashicorp.com/consul/${CONSUL_VERSION}/consul_${CO
     && rm -f consul_${CONSUL_VERSION}_linux_amd64.zip \
     && chmod +x /usr/local/bin/consul \
     && /usr/local/bin/consul -version
-    
+
 # Install mozilla/sops and AGE
 RUN dnf install -y https://github.com/mozilla/sops/releases/download/v${SOPS_VERSION}/sops-${SOPS_VERSION}-1.x86_64.rpm \
     && wget -q -O /tmp/age.tar.gz https://github.com/FiloSottile/age/releases/download/v${AGE_VERSION}/age-v${AGE_VERSION}-linux-amd64.tar.gz \
@@ -155,15 +155,11 @@ COPY Gemfile Gemfile.lock $GEM_HOME/
 RUN chown -R 1001:0 $GEM_HOME \
     && chmod -R g+rw $GEM_HOME \
     && ls -lisa /home/jenkins $GEM_HOME
-    
-RUN ls $JAVA_HOME/lib/security/cacerts \
-    && chown 1001:0 $JAVA_HOME/lib/security/cacerts \
-    && chmod g+w $JAVA_HOME/lib/security/cacerts
 
 # setup ruby env and bundler gems
 # RUBY https://syslint.com/blog/tutorial/how-to-install-ruby-on-rails-with-rbenv-on-centos-7-or-rhel-7/
 RUN cd /opt \
-    && umask 0002 \ 
+    && umask 0002 \
     && git clone https://github.com/rbenv/rbenv.git /opt/rbenv \
     && echo 'export PATH="/opt/rbenv/shims:/opt/rbenv/bin:$PATH"' >> ~/.bash_profile \
     && echo 'eval "$(rbenv init -)"' >> ~/.bash_profile \

